webapp/channels/src/components/admin_console/license_settings/modals/upload_license_modal.tsx

```diff
@@ -110,7 +110,7 @@ const UploadLicenseModal = (props: Props): JSX.Element | null => {
         <>
             <div className='content-body'>
                 <div className='svg-image'>
-                    <WomanArmOnTable
+                    <UploadLicenseSvg
                         width={200}
```

Comment:
```
width={151}
height={103}
```

Also, the `margin-top: -20px` that is applied to the `svg-image` class in this context should be removed and have `margin-bottom: 15px` instead.

Reply:
Fixed